apps/web/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "tsconfig/nextjs.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "pages/_app.tsx"],
```

Comment:
I am not that it is needed (`"**/*.tsx"` should already include `"pages/_app.tsx"`)

Reply:
Fixed